### ISSUE TITLE
App fails to deploy on Summer '20 org

### ISSUE THREAD

**Issue by pozil** [trailheadapps/ebikes-lwc#214]
When following the install instructions, `sfdx force:source:push` fails with the following error:
```
Error  force-app/main/default/experiences/E_Bikes1.site-meta.xml         An unexpected error occurred. Please include this ErrorId if you contact support: 845521647-21223 (1302355068)
ERROR running force:source:push:  Push failed.
```

**Comment by pozil** [comment]
We've identified an issue on the Salesforce Platform with the E-Bikes Community packaging in Summer '20.
We are working on a workaround.

**Comment by pozil** [comment]
Product teams are aware of the issue and have been working on a fix. Waiting for an ETA.

**Comment by schandlergarcia** [comment]
We have put in place a fix for the project deployment, but this still has not fixed the larger issue with ExperienceBundle Deployment. Check out the PR [here](https://github.com/trailheadapps/ebikes-lwc/pull/215)

1. The `force-app/main/default/experiences` has been moved to `mdapiDeploy/unpackaged/experiences` and the `package.xml` has been updated to include a reference to the `ExperienceBundle` metadata type. This must be listed at the top of the file as the `NavigationItems` also depend on the community.  

2. The aura component `forceCommunity:themeProfileMenu` has been updated to `forceCommunity:customizableUserProfileMenu` in the community `view` metadata as the `themeProfileMenu` no longer supports all flexipage types. 

3. *updated* We have added the community profile deployment into the `mdapiDeploy` folder and added it to the `package.xml` as the community needs to exist before the creation of the profile. 

The solution requires the core org functionally to be deployed before the experience bundle can be deployed on top of the existing org. 

There is currently a fix in progress (W-7823334), we will update the issue accordingly.
